### PR TITLE
Lucene linguistics

### PR DESCRIPTION
**Pull request by dainiusjocas**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by kkraune** [comment]
The post is awesome! I made some minor suggestions in the (other) shared doc for your consideration, feel free to add. let me know when reviewed by you, and we can publish :-)

File: _drafts/lucene-linguistics.md [hunk 1]
```diff
@@ -18,12 +18,12 @@ search applications still need tricks like filtering, faceting, phrase matching,
...
+The application over the years has grown to support multiple languages and for each  we have crafted custom Elasticsearch [analyzers](https://www.elastic.co/guide/en/elasticsearch/reference/8.9/specify-analyzer.html) with dictionaries for synonyms, stopwords, etc.
 Vespa has a different approach towards lexical search than Elasticsearch, and we were researching ways to transfer all that accumulated knowledge without doing the “Big Bang” migration.
 
-And here comes a chat with Jo Kristian Bergum on the sunny roof terrace at the Berlin Buzzwords 2023 conference.
-Among other things I’ve asked if it is technically possible to implement a Vespa Linguistics component on top of the Apache Lucene library.
-With Jo’s encouragement, I’ve got to work and in the same evening there was a working proof of concept.
+And here comes a part with a chat with the legend himself, [Jo Kristian Bergum](https://twitter.com/jobergum), on the sunny roof terrace at the Berlin Buzzwords 2023 conference.
```
Comment (kkraune) [inline _drafts/lucene-linguistics.md:24]: :-)

File: _drafts/lucene-linguistics.md [hunk 2]
```diff
@@ -121,27 +128,40 @@ This option works best if you’re already comfortable with Lucene.
   </config>
 </component>
 ```
-Caption: Lucene linguistics component configuration.
+
+The above analyzer uses the `standard` tokenizer, then `stop` token filter loads stopwords from the `en/stopwords.txt` file that must be placed in your application package under the `linguistics` directory; and then the `englishMinimalStem` is used to stem tokens.
+
+### Component registry
+
+The Lucene linguistics takes in an ComponentRegistry of the `Analyzer` class.
```
Comment (kkraune) [inline _drafts/lucene-linguistics.md:136]:
```suggestion
The Lucene linguistics takes in a ComponentRegistry of the `Analyzer` class.
```

**Review by kkraune — APPROVED** [review]
Better!

File: _drafts/lucene-linguistics.md (hunk 1 above)
Comment (kkraune) [reply _drafts/lucene-linguistics.md:24]: legend ...

Comment (kkraune) [reply _drafts/lucene-linguistics.md:24]: i like this - sorry for silly notes here

**Comment by dainiusjocas** [comment]
Added a screenshot

File: _drafts/lucene-linguistics.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+title: "Introducing Lucene Linguistics"
+author: dainiusjocas
+categories: []
+tags: [linguistics, bm25, stemming]
+image: assets/2023-10-16-lucene-linguistics/search-box.png
```
Comment (kkraune) [inline _drafts/lucene-linguistics.md:7]:
```suggestion
image: assets/2023-10-16-lucene-linguistics/search-box.png
date: '2023-10-16'
```

File: _drafts/lucene-linguistics.md [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+skipimage: true
```
Comment (kkraune) [inline _drafts/lucene-linguistics.md:8]:
```suggestion
skipimage: false
```

**Comment by kkraune** [comment]
Can you move the file to _posts/2023-10-16-lucene-linguistics.md - and with the above minor changes, we are good to go

**Pull request merged**